### ISSUE TITLE
Unable to parse a document

### ISSUE THREAD

**Issue by Kolkir** [plastex/plastex#128]
Hi, I'm unable to parse the simple document:

```
%-Document class----
\documentclass[a4paper, 12pt]{article}
\usepackage{minted}

\begin{document}

Some text

\section{Sample}\label{sec:sample}

Some text \textbf{Hello}

\begin{minted}[breaklines]{cpp}
void func(const std::string& image_file, long target_dim) {
...
}
\end{minted}


Some text $a+1$ with formula

\end{document}
```

The source code I used looks like this one:
```
import argparse
from plasTeX.TeX import TeX


def main():
    parser = argparse.ArgumentParser()
    parser.add_argument("filename", help="tex file name for converting")
    parser.add_argument("-o", "--output", help="output file name")
    args = parser.parse_args()

    if args.filename:
        with open(args.filename) as file:
            tex = TeX()
            tex.input(file)
            doc = tex.parse()
            print(doc)


if __name__ == "__main__":
    main()
```

The error I got is:
```
WARNING: Cannot find unidecode lib. Expect issues with index sorting
ERROR: An error occurred while building the document object in <tokens> on
   line 1 (generator raised StopIteration)
ERROR: Error while reading argument "options" of documentclass in test.tex
   on line 2 (generator raised StopIteration)
ERROR: Error while expanding "documentclass" in test.tex on line 2
   (generator raised StopIteration)
ERROR: An error occurred while building the document object in test.tex on
   line 2 (generator raised StopIteration)
Traceback (most recent call last):
  File "/home/kirill/development/tex2text/venv/lib/python3.7/site-packages/plasTeX/Tokenizer.py", line 385, in __iter__
    token = next(charIter)
StopIteration

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/kirill/development/tex2text/tex2text.py", line 20, in <module>
    main()
  File "/home/kirill/development/tex2text/tex2text.py", line 15, in main
    doc = tex.parse()
  File "/home/kirill/development/tex2text/venv/lib/python3.7/site-packages/plasTeX/TeX.py", line 435, in parse
    for item in tokens:
  File "/home/kirill/development/tex2text/venv/lib/python3.7/site-packages/plasTeX/TeX.py", line 46, in __next__
    return self._next()
  File "/home/kirill/development/tex2text/venv/lib/python3.7/site-packages/plasTeX/TeX.py", line 343, in __iter__
    tokens = obj.invoke(self)
  File "/home/kirill/development/tex2text/venv/lib/python3.7/site-packages/plasTeX/Base/LaTeX/Packages.py", line 36, in invoke
    a = self.parse(tex)
  File "/home/kirill/development/tex2text/venv/lib/python3.7/site-packages/plasTeX/__init__.py", line 468, in parse
    **arg.options)
  File "/home/kirill/development/tex2text/venv/lib/python3.7/site-packages/plasTeX/TeX.py", line 757, in readArgumentAndSource
    toks, source = self.readGrouping(spec, expanded, parentNode=parentNode)
  File "/home/kirill/development/tex2text/venv/lib/python3.7/site-packages/plasTeX/TeX.py", line 909, in readGrouping
    toks = self.expandTokens(toks, parentNode=parentNode)
  File "/home/kirill/development/tex2text/venv/lib/python3.7/site-packages/plasTeX/TeX.py", line 406, in expandTokens
    out = tex.parse(frag)
  File "/home/kirill/development/tex2text/venv/lib/python3.7/site-packages/plasTeX/TeX.py", line 435, in parse
    for item in tokens:
  File "/home/kirill/development/tex2text/venv/lib/python3.7/site-packages/plasTeX/TeX.py", line 46, in __next__
    return self._next()
  File "/home/kirill/development/tex2text/venv/lib/python3.7/site-packages/plasTeX/TeX.py", line 322, in __iter__
    token = next()
  File "/home/kirill/development/tex2text/venv/lib/python3.7/site-packages/plasTeX/TeX.py", line 266, in itertokens
    t = next(inputs[-1][-1])
RuntimeError: generator raised StopIteration

Process finished with exit code 1
```

**Comment by PatrickMassot** [comment]
How did you install plasTeX? The version which is currently on PyPi is not compatible with python3.7. In order to use Python3.7 you need to clone the repository and pip install from there.

**Comment by Kolkir** [comment]
Hi @PatrickMassot, yes, I've used PyPi to install plasTeX. I will try to install it from sources as you advise.

**Comment by PatrickMassot** [comment]
This issue was fixed by releasing plasTeX 2.1.
